azure-pipelines.yml

```diff
@@ -0,0 +1,309 @@
+      Select-String $(host) .\MsSetup.inc
+      Select-String $(pdo_sqlsrv_db) .\MsSetup.inc
+    displayName: 'Update connection credentials'
+    condition: false
+
+  - powershell: |
+      $client = New-Object Net.WebClient
+      $client.DownloadFile('https://download.microsoft.com/download/E/6/B/E6BFDC7A-5BCD-4C51-9912-635646DA801E/en-US/msodbcsql_17.3.1.1_x64.msi', 'msodbcsql_17.3.1.1_x64.msi')
+      $client.DownloadFile('https://download.microsoft.com/download/D/5/E/D5EEF288-A277-45C8-855B-8E2CB7E25B96/x64/msodbcsql.msi', 'msodbcsql_13.1.msi')
+      $client.DownloadFile('https://download.microsoft.com/download/4/C/C/4CC1A229-3C56-4A7F-A3BA-F903C73E5895/EN/x64/MsSqlCmdLnUtils.msi', 'MsSqlCmdLnUtils.msi')
+      dir *.msi
+    displayName: 'Download ODBC msi and sql tools msi'
+    condition: false
+
+  - script: |
+      msiexec /i "msodbcsql_17.3.1.1_x64.msi" /q IACCEPTMSODBCSQLLICENSETERMS=YES ADDLOCAL=ALL
+      reg query "HKLM\SOFTWARE\ODBC\odbcinst.ini\ODBC Driver 17 for SQL Server"
+      dir %WINDIR%\System32\msodbcsql*.dll
+    displayName: 'Install ODBC driver'
+    condition: false
+
+  # TOFIX: Install ODBC 13.1 because of SQLCMD 15 installation bug -- this step should be removed later
+  - script: msiexec /i "msodbcsql_13.1.msi" /q IACCEPTMSODBCSQLLICENSETERMS=YES ADDLOCAL=ALL
+    condition: false
+
+  # FOR SOME REASON the set up did not set the PATH 
+  - script: |
+      msiexec /i "MsSqlCmdLnUtils.msi" /qn IACCEPTMSSQLCMDLNUTILSLICENSETERMS=YES
+    displayName: 'Install SQL command line utilities version 15'
+    condition: false
+
+  - powershell: |
+      $client = New-Object Net.WebClient
+      $client.Headers.Add("user-agent", "azure pipeline build")
+      $client.DownloadFile("https://windows.php.net/downloads/releases/sha256sum.txt", "sha256sum.txt")
+      $env:VERSION=type sha256sum.txt | where { $_ -match "php-($(phpVersion)\.\d+)-src" } | foreach { $matches[1] }
+      Write-Host "Latest PHP $(phpVersion) is ${env:VERSION}" 
+      cd $(Build.SourcesDirectory)/buildscripts/
+      python builddrivers.py --PHPVER=${env:VERSION} --ARCH=x64 --THREAD=nts --SOURCE=$(Build.SourcesDirectory)/source --TESTING --NO_RENAME
+      cp php-sdk\phpdev\vc15\x64\php-${env:VERSION}-src\run-tests.php $(Build.SourcesDirectory)\test\functional\sqlsrv
+      cp php-sdk\phpdev\vc15\x64\php-${env:VERSION}-src\run-tests.php $(Build.SourcesDirectory)\test\functional\pdo_sqlsrv
+      dir *sqlsrv*.dll
+      cp *sqlsrv*.dll C:\tools\php\ext\
+    displayName: 'Build drivers for the latest version of PHP $(phpVersion)'
+  
+  - script: |
+      echo extension=php_sqlsrv.dll >> C:\tools\php\php.ini
+      echo extension=php_pdo_sqlsrv.dll >> C:\tools\php\php.ini
+      php --ri sqlsrv
+      php --ri pdo_sqlsrv

```

Comment:
Is this the correct php? Does the VM have php already installed?

Reply:
Yes, it has php already installed, as in the Linux agent